android/gradle.properties

```diff
@@ -1,4 +1,3 @@
-android.useDeprecatedNdk=true
```

Comment:
🙌

Reply:
Frankly I didn't do anything but respond to a warning stating that _this option has been removed and doesn't have any effect_. 😅